src/alire/alire-utils-user_input.adb

```diff
@@ -15,6 +15,17 @@ package body Alire.Utils.User_Input is
       No     => 'N',
       Always => 'A');
 
+   type String_Access is access constant String;
```

Comment:
Instead of using pointers here you can define and expression function to return the String:
```ada
   function Answer_String (Kind : Answer_Kind) return String
   is (case Kind is
       when Yes    => "yes",
       when No     => "no",
       when Always => "always");
```

Reply:
That looks a lot cleaner to me, too. I replaced my code with your function.